usaspending_api/api_contracts/contracts/v2/disaster/agency/spending.md

```diff
@@ -0,0 +1,167 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agencies Spending Disaster/Emergency Funding [/api/v2/disaster/agency/spending/]
+
+This endpoint provides insights on the Agencies which received disaster/emergency funding per the requested filters.
+
+## POST
+
+Returns spending details of Agencies receiving supplemental funding budgetary resources
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes
+        + `filter` (required, Filter, fixed-type)
+        + `spending_type` (required, enum[string], fixed-type)
+            Toggle if the outlay and obligation response values are total or only from awards only from awards
+            + Members
+                + `total`
+                + `award`
+        + `pagination` (optional, Pagination, fixed-type)
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `results` (required, array[Result], fixed-type)
+        + `pagination_metadata` (required, PageMetadata, fixed-type)
+
+
+    + Body
+
+            {
+                "results": [
+                    {
+                        "id": 43,
```

Comment:
`id` is listed as a string in the `Result` data structure

Reply:
fixed